lib/ecommerce/resources/adjustment_order.rb

```diff
@@ -0,0 +1,85 @@
+module Ecommerce
+  module Resources
+    #
+    # A wrapper to Ecommerce adjusments orders API
+    #
+    # [API]
+    #   Documentation: http://myfreecomm.github.io/passaporte-web/ecommerce/api/index.html
+    #
+    class AdjustmentOrder < Base
+      attr_reader :url, :amount, :description, :valid_until, :valid_from
+
+      #
+      # Adjustment order API does not return the ID field
+      #
+      def id
+        url.split('/')[8].to_i
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.